lib/Sema/TypeCheckAttr.cpp

```diff
@@ -6861,7 +6861,8 @@ void AttributeChecker::visitNonisolatedAttr(NonisolatedAttr *attr) {
       // 'nonisolated' can not be applied to mutable stored properties unless
       // qualified as 'unsafe'.
       if (var->supportsMutation() && !attr->isUnsafe()) {
-        diagnoseAndRemoveAttr(attr, diag::nonisolated_mutable_storage);
+        diagnoseAndRemoveAttr(attr, diag::nonisolated_mutable_storage)
+            .fixItInsertAfter(attr->getRange().End, "(unsafe)");
```

Comment:
One thing that I particular would enjoy review feedback on is adding a second fixit to this diagnostic that can change `nonisolated` to `nonisolated(unsafe)`.
1. I want to confirm that my understanding is correct that attaching multiple mutually-exclusive fixits is an okay approach and that the user would choose one or the other, but not both?
2. I want to know if folks have opinions about even offering an "unsafe" option as one of the fixits?

Reply:
Thanks for confirming my first question, and for the second, I have added a note as you suggested, and `nonisolated(unsafe)` should already be the second fixit.